registry/client.go

```diff
@@ -1061,17 +1064,23 @@ func (c *client) GetActivePoolStake(blockNumber, poolID *big.Int) (*big.Int, err
 	return big.NewInt(0).Add(poolStake, delegatorPoolStake), nil
 }
 
-func (c *client) TotalScannersRegistered(blockNumber, poolID *big.Int) (*big.Int, error) {
+func (c *client) GetAllocatedStakePerManaged(blockNumber, poolID *big.Int) (*big.Int, error) {
 	contracts := c.Contracts()
-	if contracts.ScannerPoolReg == nil {
-		return nil, ErrContractNotReady
-	}
 
 	var opts bind.CallOpts
 	if c.opts != nil {
 		opts = *c.opts
 	}
 	opts.BlockNumber = blockNumber
 
-	return contracts.ScannerPoolReg.TotalScannersRegistered(&opts, poolID)
+	return contracts.StakeAllocator.AllocatedStakePerManaged(&opts, SubjectTypeScannerPool, poolID)
+}
+
+func (c *client) TotalScannersRegistered(poolID *big.Int) (*big.Int, error) {
```

Comment:
I wasn't sure: Do we still need this method?

Reply:
Techinically no, do you prefer I get rid of it?

Reply:
Maybe let's remove it for now (and generate the mocks again). It's better to keep the interface smaller. :)